tests/Samples.Server.Tests/BaseTest.cs

```diff
@@ -39,10 +58,20 @@ public virtual void Dispose()
         {
             Client.Dispose();
             Server.Dispose();
+
+#if NETCOREAPP2_2
+#else
```

Comment:
can be changed to one line #if NETCOREAPP3_0

Reply:
Yea I've done this so that we can avoid changing it in the future, e.g. when 4_0 comes out. Assuming we can use the same code we shouldn't have to do anything other than deal with package versions in csproj files.

Reply:
Guess I can do `!NETCOREAPP2_2` as we prob won't add any other older targets. Done.